src/client/src/apps/MainRoute/widgets/blotter/blotterFields.ts

```diff
@@ -0,0 +1,11 @@
+export const STATUS_INDICATOR = 'statusIndicator'
```

Comment:
added this because I want all field names to be available without the need to do deep import (/widgets/components/blotterUtils)